docs/air-pool/create-airpool-erc1155.md

```diff
@@ -1,34 +1,38 @@
 ---
-sidebar_position: 4
+sidebar_position: 5
 ---
 
-# Creating ERC1155 Air Pool
+# Creating ERC1155 AirPool
 
 This article focuses on how to create your first ERC 1155 reward pool on AirLyft.
 
-- Login to AirLyft and once you are on the dashboard, please click on AirPool icon as shown in the screenshot below.
+- There are two ways to create ERC 1155 AirPool. You can either create them from the AirPool tab, or you can click on **Create New Pool** while adding a reward to your campaign.
```

Comment:
Create two subheadings to make it easier to read
1. Creating a pool within campaign
2. Creating a pool through Air Pools page
Please do this for all .. even air tokens